src/interfaces/banner.ts

```diff
@@ -79,6 +79,11 @@ export type Action =
       actionName: 'close-swap-and-bridge'
       meta: { activeRouteId: number }
     }
+  | {
+      label: 'Close'
+      actionName: 'hide-activity-banner'
+      meta: { timestamp: number; addr: string; network: string; isReject: boolean }
```

Comment:
Three terms are used for the same action:
```
close- in the label ('Close')
reject- in meta (isReject: boolean)
hide- in the action (ACTIVITY_CONTROLLER_HIDE_BANNER)
```
please choose one term and stick to it for clarity or one term for internal use and one for the UI. For example `hide` in the code and `Close` in the UI